File: src/main/java/com/procurement/access/model/dto/tender/Framework.java
package com.procurement.access.model.dto.tender;

import lombok.Data;

import java.util.List;

@Data
public class Framework {
    public Boolean isAFramework;
    public String typeOfFramework;
    public Integer maxSuppliers;
    public String exceptionalDurationRationale;
    public List<String> additionalBuyerCategories;
    public String description;
}

File: src/main/kotlin/com/procurement/access/model/dto/ocds/Framework.kt
package com.procurement.access.model.dto.ocds

import com.fasterxml.jackson.annotation.JsonCreator
import com.fasterxml.jackson.annotation.JsonProperty
import javax.validation.constraints.NotNull

data class Framework @JsonCreator constructor(

        @field:NotNull
        @get:JsonProperty("isAFramework")
        val isAFramework: Boolean
)